operatorcourier/log_util.py

```diff
@@ -0,0 +1,67 @@
+import operatorcourier.log_dict_util as log_dict_util
+import logging.config
+from io import StringIO
+import json
+
+class log_util():
+    class log_util():
+        """
+        Wraps the Python logging module's logger object to ensure that all
+        logging is formatted into a json dictionary.
+        """
+        logger = None
+        
+        def __init__(self):
+            self._log_dict = dict(
+                    info=[],
+                    debug=[],
+                    warning=[],
+                    error=[],
+                    critical=[]
+                )
+            #logging.config.fileConfig("./config/logging.ini")
+            logging.basicConfig(format='%(levelname)s:%(message)s', level=logging.DEBUG)
+            self._logger = logging.getLogger()
+            
+
+
+        def debug(self, message, *args, **kwargs):
+            log_dict_util.debug(self._log_dict, message % args)
+            self._logger.debug(message, *args, **kwargs)
+
+        def info(self, message, *args, **kwargs):
+            log_dict_util.info(self._log_dict, message % args)
+            self._logger.info(message, *args, **kwargs)
+
+        def warning(self,  message, *args, **kwargs):
+            """warning will log atakes a string appends it to a JSON object.
+
+            :param message: string to add to warning dict
+            """
+            log_dict_util.warning(self._log_dict, message % args)
+            self._logger.warning(message, *args, **kwargs)
+
+        def error(self, message, *args, **kwargs):
+            """error takes a string appends it to a JSON object.
+
+            :param message: string to add to error dict
+            """
+            log_dict_util.error(self._log_dict, message % args)
+            self._logger.error(message, *args, **kwargs)
+
+        def critical(self, message, *args, **kwargs):
+            log_dict_util.info(self._log_dict, message % args)
+            self._logger.critical(message, *args, **kwargs)
+
+        def fPrint(self, log_file):
+            if log_file:
```

Comment:
Isn't this quite unexpected, why even call this method when log_file is not specified at first place?